src/main/java/com/spotify/reaper/storage/CassandraStorage.java

```diff
@@ -142,7 +153,7 @@ public boolean isStorageConnected() {
   @Override
   public boolean addCluster(Cluster cluster) {
     try {
-      session.execute(insertClusterPrepStmt.bind(cluster.getName(), cluster.getPartitioner(), cluster.getSeedHosts()));
+      session.executeAsync(insertClusterPrepStmt.bind(cluster.getName(), cluster.getPartitioner(), cluster.getSeedHosts()));
```

Comment:
It looks like you're not waiting for any async write to return and exit the method immediatly.
This will lead to potential silent failures and chained calls being inconsistent (we may try to read something that isn't yet written to all replicas).

Is it on purpose or did you forget the `.get()` after the `executeAsync` calls ? 
Also, in the case we have a single call in the method, we may be better just running a synchronous `execute`.